enricher/fabric8/src/main/java/io/fabric8/maven/enricher/fabric8/SpringBootHealthCheckEnricher.java

```diff
@@ -42,6 +44,12 @@
     private static final String SCHEME_HTTPS = "HTTPS";
     private static final String SCHEME_HTTP = "HTTP";
 
+    private enum Config implements Configs.Key {
+        timeoutSeconds {{ d = "1"; }};
```

Comment:
Ah yeah using what is the default in k8s would be better. 

Also I think this functionality about the timeout is needed in the other health checkers for wildfly-swarm, vertx etc. But that can be done in another PR